### ISSUE TITLE
Mirroring illuminate/support archives more packages than necessary

### ISSUE THREAD

**Issue by jcbwlkr** [composer/satis#139]
TLDR: Trying to mirror package with no dependencies pulls down tons of packages.

I'm pretty sure this is a bug in Satis but I might be missing something.

I am trying to mirror my dependencies from Packagist. When I require `"illuminate/support": "v4.1.25"` then run `build` I get a couple gigabytes worth of Laravel, Doctrine, and related packages. Looking on Packagist and GitHub it seems that `illuminate/support` has no dependencies. It does have dev dependencies but I have `"require-dev-dependencies": false`.

My `satis.json`:

``` json
{
  "name": "My Company",
  "homepage": "http://url",
  "repositories": [
    { "type": "composer", "url": "https://packagist.org" }
  ],

  "require": {
    "illuminate/support": "v4.1.25"
  },

  "require-dependencies": true,
  "require-dev-dependencies": false,

  "archive": {
    "directory": "dist",
    "format": "tar"
  }
}
```

The command I'm running to build is:

```
php bin/satis -n build satis.json web
```

A small sample of some of the packages being downloaded:

```
amaly-laravel-framework-f53b06c5bc68bef089dca064607ce471dce7d8ab-zip-bc67b2.tar
amaly-laravel-framework-f8a19f219ba4b70d0aaa95f029c4d2644bb47f54-zip-13968e.tar
anlutro-curl-407698f2f61a33393b2ab8ad4c0cfcf6e170dc18-zip-372b07.tar
cilex-console-service-provider-25ee3d1875243d38e1a3448ff94bdf944f70d24e-zip-e62a74.tar
classpreloader-classpreloader-1a50f7945b725ff2c60f234e51407d1d6e7c77c5-zip-844ef8.tar
classpreloader-classpreloader-2c9f3bcbab329570c57339895bd11b5dd3b00877-zip-b98924.tar
d11wtq-boris-a8ad0ffc07bca496c94ef1c2a921b5f913fd0795-zip-628f0e.tar
d11wtq-boris-b3ac553863d3445e1545096f9f8a97147659e01b-zip-cea9b5.tar
doctrine-annotations-d2bb4eaf82f74fd6964503b4826d3963e134e400-zip-0b3b13.tar
doctrine-annotations-fae359b3efd908e407a0105ff8956b5c94ddca8e-zip-627787.tar
doctrine-cache-2c9761ff1d13e188d5f7378066c1ce2882d7a336-zip-16955d.tar
doctrine-collections-b1e6852b5e49e94ac818feca3b1a8689daefbffc-zip-a07ec5.tar
doctrine-collections-b99c5c46c87126201899afe88ec490a25eedd6a2-zip-1be3b8.tar
doctrine-common-17e774007b98beb2e253e645260e0f9c32f4c936-zip-da0ce5.tar
```

Thanks for looking, and thanks for a great tool. Let me know if I can provide any additional information.
